dashboard/test/lib/standards_autocomplete_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'test_helper'
+
+class StandardsAutocompleteTest < ActiveSupport::TestCase
+  # We rely on fulltext indices to be used in this test.
+  #
+  # Fulltext indexes are only updated when database changes are committed,
+  # which means they will not be updated within a transactional test case:
+  # https://dev.mysql.com/doc/refman/8.0/en/fulltext-restrictions.html
+  # This makes it hard to create a standard via factory and then search for it.
+  #
+  # Currently, all standards from dashboard/config/standards/ are seeded in unit
+  # tests, and these unit tests search against those standards.
```

Comment:
generally speaking, it is bad for unit tests to depend on production data such as what's in dashboard/config/standards. However, standards are relatively static compared to vocab or resources, and can currently only be modified by engineers, so it seems like the risk of test failures due to updating our standards is relatively low. 

If this becomes a problem, we can either:
1. stop seeding all standards in unit tests, and switch to using test fixtures like we do for resources and vocab, or
2. try harder to make unit tests that create standards/frameworks outside of database transactions (I tried this initially but was unsuccessful)